src/assets.zig

```diff
@@ -260,7 +260,9 @@ pub fn readAssets(
 	readAllObjFilesInAddonsHashmap(externalAllocator, addons, addonNames, "models", models);
 }
 
-fn registerItem(assetFolder: []const u8, id: []const u8, zon: ZonElement) !*items_zig.BaseItem {
+fn registerItem(assetFolder: []const u8, id: []const u8, zon: ZonElement) !void {
+	if(items_zig.hasRegistered(id)) return;
```

Comment:
This check should only be made when trying to register new items, not when registering items from the palette to avoid problems with duplicate entries (produced by migrating multiple items onto one id).